lib/repl.js

```diff
@@ -501,7 +501,9 @@ function REPLServer(prompt,
   self.writer = options.writer || exports.writer;
 
   if (options.useColors === undefined) {
-    options.useColors = self.terminal;
+    options.useColors = self.terminal && (
+      typeof self.outputStream.getColorDepth === 'function' ?
+        self.outputStream.getColorDepth() > 2 : true);
```

Comment:
Shouldn't this be `> 1`?

Reply:
Good catch while it won't have any actual impact (the return values are either `1`, `4`, `8` or `24`).

Reply:
I got that check from `lib/internal/console/constructor.js:256-258`, should I change the check there too in this PR?

Reply:
@wlodzislav I think it’s okay to leave that for a separate PR, but here it would be good to replace it with `> 1` before this is being merged.

Reply:
I fixed this while landing.